examples/osp-flatcar.yaml

```diff
@@ -378,9 +387,11 @@ spec:
 
             {{- /* Since both container runtimes are enabled/started by default in flatcar, disable the one that is not required */}}
             {{- if eq .ContainerRuntime "containerd" }}
+            systemctl stop docker
```

Comment:
Do we actually need docker if containerd is used

Reply:
Nope we don't, that's why we just stop and disable it.